lib/Alchemy/Phrasea/WorkerManager/Controller/AdminConfigurationController.php

```diff
@@ -92,6 +95,82 @@ public function infoAction(PhraseaApplication $app, Request $request)
         ]);
     }
 
+    public function infoSubtitleAction(PhraseaApplication $app)
+    {
+        /** @var WorkerJobRepository $repoWorkerPopulate */
+        $repoWorkerJob = $app['repo.worker-job'];
+
+        $workerSubtitleRunning  = $repoWorkerJob->findBy(['type' => MessagePublisher::SUBTITLE_TYPE, 'status' => WorkerJob::RUNNING]);
+        $workerSubtitleFinished = $repoWorkerJob->findBy(['type' => MessagePublisher::SUBTITLE_TYPE, 'status' => WorkerJob::FINISHED]);
+
+        $subtitlesRunning = [];
+        $subtitlesFinished = [];
+
+        /** @var WorkerJob $running */
+        foreach ($workerSubtitleRunning as $running) {
+            $data = $running->getData();
+            $data['status']   = $running->getStatus();
```

Comment:
here it's always "running"...
btw, if we **really** want to return 2 separate arrays "running" and "finished", we must be more generic :
repo->findall(), then fill arrays by type --> smaller code, no code change for new type.